extensions/wikia/AdEngine/js/video/player/porvata/floaterConfiguration.js

```diff
@@ -28,14 +25,28 @@ define('ext.wikia.adEngine.video.player.porvata.floaterConfiguration', [
 						floatingContext.preferred.width = body.scrollWidth;
 						floatingContext.preferred.height = body.scrollHeight;
 						floatingContext.floatAgain();
+						floatingContext.fireEvent('start');
+
+						if (floatingContext.isOutsideOfViewport()) {
+							callback(floatingContext);
+						}
 					});
 
 				},
 				onAttach: function (floatingContext) {
-					var elements =floatingContext.elements;
+					var elements = floatingContext.elements;
 
 					elements.ad.style.maxHeight = elements.ad.scrollHeight + 'px';
 					elements.viewport.style.removeProperty('height');
+					elements.providerContainer.style.removeProperty('height');
+				},
+				onBeforeDetach: function (floatingContext) {
+					var viewport = floatingContext.elements.viewport;
+
+					viewport.style.height = viewport.offsetHeight + 'px';
+				},
+				onDetach: function (floatingContext) {
+					floatingContext.elements.providerContainer.style.height = floatingContext.getHeight() + 'px';
```

Comment:
Because `floatingContext.getHeight()` will be incorrect. It is computed based on width and width is modified to alien mode between `beforeDetach` and `deatch` events.